src/inject/dynamic-theme/modify-css.ts

```diff
@@ -70,7 +70,7 @@ export function getModifiedUserAgentStyle(theme: Theme, isIFrame: boolean, style
     lines.push(`    color: ${modifyForegroundColor({r: 0, g: 0, b: 0}, theme)};`);
     lines.push('}');
     lines.push('a {');
-    lines.push(`    color: ${modifyForegroundColor({r: 0, g: 64, b: 255}, theme)};`);
+    lines.push(`    color: ${modifyForegroundColor({r: 0, g: 0, b: 238}, theme)};`);
```

Comment:
Why this color is changed?

Reply:
The original one introduce green, which is weird.
So looking at [Firefox](https://searchfox.org/mozilla-central/source/modules/libpref/init/StaticPrefList.yaml#1165)'s color for this. It hasn't any green same story for [Chromium](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/dom/text_link_colors.cc;l=42). Was their any reason for the custom color to make it more appropriate blue?

Reply:
There is a problem with pure blue (240 degrees in HSL). In light mode the difference is barely visible, but in dark mode it looks too dark. Increasing the lightness makes it look violet like a visited link. So the slight hue change to 220 degrees makes it look blue and more contrast. If you have a look at native dark themes for some websites, designers usually slide the hue a little bit, to make it more bright. There's also a compensation in the formula, that slides 240 degrees to 220.
<img width="365" alt="Screen Shot 2021-01-20 at 20 34 08" src="https://user-images.githubusercontent.com/9638588/105226489-54a4b300-5b60-11eb-9f4e-8425b85bd562.png">
This way we get more contrast for blue links: your suggestion becomes 220 deg, the previous value becomes 212 deg in HSL. This was an irritating issue in the past.
<img width="608" alt="Screen Shot 2021-01-20 at 20 46 31" src="https://user-images.githubusercontent.com/9638588/105226722-abaa8800-5b60-11eb-83d9-652ed616e012.png">

Reply:
Got it 👍🏽